viewer/packages/tools/src/Measurement/MeasurementGizmo.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Cognite3DViewer } from '@reveal/core';
+import * as THREE from 'three';
+
+export class MeasurementGizmo {
```

Comment:
This seems unnecessary, a simple function creating the mesh should be enough.

Also, avoid creating SphereGeometry and MeshBasicMaterial for every instance as that will cause upload to gpu and shader compile I believe.

Reply:
Wouldn't this be helpful for editing existing measurement?